shared/AppInsightsCore/src/JavaScriptSDK/W3cTraceParent.ts

```diff
@@ -1,5 +1,7 @@
-import { generateW3CId, getDocument, getPerformance, isArray, isString, strTrim } from "@microsoft/applicationinsights-core-js";
-import { ITraceParent } from "./Interfaces/ITraceParent";
+import { ITraceParent } from "../JavaScriptSDK.Interfaces/ITraceParent";
+import { generateW3CId } from "./CoreUtils";
+import { findMetaTag, findNamedServerTiming, getDocument, getPerformance } from "./EnvUtils";
```

Comment:
There is code scanning alert for unused import!

Reply:
Will check and fix those